extras/module-node/Makefile

```diff
@@ -14,3 +14,4 @@ test:
 	./test 'var ape = require("ape"); ape.module.filename === "ape.js" ? "module.filename OK" : "module.filename FAILED";'
 	./test 'var ape = require("ape"); ape.module.loaded === true && ape.wasLoaded === false ? "module.loaded OK" : "module.loaded FAILED";'
 	./test 'var ape = require("ape"); ape.__filename === "ape.js" ? "__filename OK" : "__filename FAILED";'
+	./test 'var badger = require("badger"); badger.foo === 123 && badger.bar === 234;'
```

Comment:
There's a lot of other gaps in coverage (same for other extras so it's not something being wrong here) but I wanted that basic `exports.foo = 123` to be covered because it's so central to modules.